Asterisk.2013/Asterisk.NET/Manager/Event/QueueMemberPausedEvent.cs

```diff
@@ -1,10 +1,13 @@
+using System;
+
 namespace AsterNET.Manager.Event
 {
 	/// <summary>
 	/// A QueueMemberPausedEvent is triggered when a queue member is paused or unpaused.<br/>
 	/// It is implemented in apps/app_queue.c.<br/>
 	/// Available since Asterisk 1.2
 	/// </summary>
+    [Obsolete("Removed since Asterisk 13", false)]

```

Comment:
Is it the wording that is confusing? How about:
"This class is supported only up to Asterisk 12, Please use '`QueueMemberPauseEvent`' from Asterisk 12 onward."